countrypicker/src/main/java/com/canopas/campose/countrypicker/CountrySearchView.kt

```diff
@@ -87,13 +87,14 @@ fun countrySearchView(state: ModalBottomSheetState): String {
                     keyboardType = KeyboardType.Text,
                     imeAction = ImeAction.Search
                 ),
-                keyboardActions = KeyboardActions(onDone = { focusManager.clearFocus() })
+                keyboardActions = KeyboardActions(onSearch = { focusManager.clearFocus() })
```

Comment:
Is this change going to shift focus from  select country to Mobile number field in NoLonely?
Or we can say like shift focus from select country to Next field?

Reply:
no, when we search country in search view and click on search option of keyboard then Keyboard  should  close. but not occured any event of keyboard. that's why i changed

Reply:
@cp-hardik-p Does our Nolonely's issue of keyboard will be solved after this?

Reply:
@cp-megh-l i think this issue is not related to nolonely.

Reply:
Hmm

Reply:
So on click of search button from keyboard after searching country , now keyboard will be closed right? @cp-luzi-p

Reply:
yes